Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -399,6 +402,15 @@ bool PPCSymbolDB::LoadMap(const Core::CPUThreadGuard& guard, const std::string&
     if (name[strlen(name) - 1] == '\r')
       name[strlen(name) - 1] = 0;
 
+    // Split the current name string into separate parts, and get the object name
+    // if it exists.
+    const std::vector<std::string> parts = SplitString(name, '\t');
+    size_t num_parts = parts.size();
```

Comment:
This should also be `const` or used directly in the ternary if not used elsewhere (see my [previous suggestion](https://github.com/dolphin-emu/dolphin/pull/13113#discussion_r1826559533)).